# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../corona/gcc43.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur corona-1.0.2-orig/src/Convert.cpp corona-1.0.2/src/Convert.cpp
--- corona-1.0.2-orig/src/Convert.cpp	2009-02-08 22:37:13.000000000 +0000
+++ corona-1.0.2/src/Convert.cpp	2009-02-08 22:38:48.000000000 +0000
@@ -9,7 +9,7 @@
 #include "Debug.h"
 #include "SimpleImage.h"
 #include "Utility.h"
-
+#include <cstring>
 
 namespace corona {
 
diff -Nur corona-1.0.2-orig/src/OpenTGA.cpp corona-1.0.2/src/OpenTGA.cpp
--- corona-1.0.2-orig/src/OpenTGA.cpp	2009-02-08 22:37:13.000000000 +0000
+++ corona-1.0.2/src/OpenTGA.cpp	2009-02-08 22:44:00.000000000 +0000
@@ -3,6 +3,7 @@
 #include "Open.h"
 #include "SimpleImage.h"
 #include "Utility.h"
+#include <cstring>
 
 
 namespace corona {
